### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by dpurnomo** [alx-plugins/better-fn#7]
Able to install but not able to load the plugin.

**Comment by aidenlx** [comment]
Yes, it's not tested on mobile, so I've removed the mobile flag in the latest release to avoid being searched on mobile and will add it back when it's fully tested.

**Comment by leffsue** [comment]
这个插件太强了…请问移动端还有消息么~
